pages/donasi.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import { GetStaticProps } from "next";
+import { NextSeo } from "next-seo";
+import { InternalPageContent } from "~/components/layout/internal-page-content";
+import { DonationCard } from "~/components/donasi/donation-card";
+import { BackButton } from "~/components/layout/back-button";
+import { Page } from "~/components/layout/page";
+import { PageHeader } from "~/components/layout/page-header";
+import { Container } from "~/components/ui/container";
+import informasiDonasi, {
+  Donation,
+  DonationDetail,
+} from "~/lib/content/informasi-donasi";
+
+type InformasiDonasiProps = {
+  informasiDonasi: Donation;
+};
+
+const meta = {
+  title: `Donasi dan Penggalangan Dana`,
+  description: `Kumpulan informasi mengenai donasi, nomor rekening, dan lainnya yang relevan dengan bantuan COVID-19.`,
+};
+
+export default function DonasiPage(props: InformasiDonasiProps) {
+  return (
+    <div>
+      <Page>
+        <NextSeo
+          description={meta.description}
+          openGraph={{ title: meta.title, description: meta.description }}
+          title={meta.title}
+        />
+        <InternalPageContent>
+          <Container className="space-y-2">
+            <PageHeader
+              backButton={<BackButton href="/" />}
+              breadcrumbs={[
+                {
+                  name: "Donasi",
+                  href: "/donasi",
+                  current: true,
+                },
+              ]}
+              title="Donasi dan Penggalangan Dana"
+            />
+            <div className="p-4 overflow-hidden rounded-md space-y-6 bg-white">
+              <div>
+                <div className="text-sm text-gray-600 mb-4">
+                  {meta.description}
```

Comment:
Put this on the `description` prop on `PageHeader` instead. This is added on #574.

Reply:
ahh got it. Thanks! 🙇